src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -168,3 +168,15 @@ Every pull request needs these labels so we can see where our contributions come
   - `from_external`: A user opened it in the repo OR it came in through #customer-feedback process.
   - `from_tw`: One of us created it (unless we were passing along #customer-feedback). If the PR fixes an external issue, label it as `from_tw` since the work was done by a tech writer.
 
+## Docs issues [#docs-issues]
+
+Docs issues is a docs team program that enlists the help of product teams in solving documentation and product issues. The goal is to get product team expertise in solving issues filed by other relics and our users. Docs issues candidates are issues that would be difficult for a docs hero to chase down or would benefit from deep engineering expertise. Examples of good candidates are code examples, requirements sections, and NRQL queries.
```

Comment:
"relics" s/be "Relics"
This paragraph is fine although every sentence in it uses the verb "to be" :-)